backend/lib/use-cases/acms-orders/acms-orders.ts

```diff
@@ -42,52 +47,17 @@ export type AcmsTransformationResult = {
   leadCaseId: string;
   childCaseCount: number;
   success: boolean;
-};
-
-export type AcmsAggregate = {
-  successful: {
-    leadCaseCount: number;
-    childCaseCount: number;
-  };
-  failed: {
-    leadCaseIds: string[];
-    leadCaseCount: number;
-    childCaseCount: number;
-  };
-};
-
-export type AcmsPageReport = AcmsAggregate & {
-  predicateAndPage: AcmsPredicateAndPage;
-};
-
-export type AcmsPartitionReport = AcmsAggregate & {
-  predicate: AcmsPredicate;
+  error?: CamsError;
```

Comment:
Do we want to add errors to the migration results we store in the "fail" queue??